src/main/java/org/candlepin/service/model/CloudCheckInEvent.java

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright (c) 2009 - 2024 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.service.model;
+
+import org.candlepin.model.Consumer;
+import org.candlepin.model.ConsumerCloudData;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * An event where a cloud consumer has updated their last check-in time stamp.
+ */
+public class CloudCheckInEvent implements AdapterEvent {
+
+    private static final String SYSTEM_UUID_KEY = "systemUuid";
+    private static final String CHECK_IN_KEY = "checkIn";
+    private static final String PROVIDER_KEY = "cloudProviderId";
+    private static final String ACCOUNT_KEY = "system-uuid";
```

Comment:
instead of `system-uuid` this should be `cloudAccountId`

Reply:
Updated